### ISSUE TITLE
Crashes on examples with "TypeError: Cannot read property 'message' of undefined"

### ISSUE THREAD

**Issue by kot-lex** [samccone/drool#39]
Hello! 
I'm trying to run drool test example the following way:
`node test/analysis/gc-thrash.js`

but constantly get an exception: 
```
TypeError: Cannot read property 'message' of undefined
    at lib/index.js:56:29

```

Node v8.9.1
ChromeDriver 2.33.506106

**Comment by zobier** [comment]
This is fixed by https://github.com/samccone/drool/pull/35

**Comment by samccone** [comment]
released as v0.6.0
